src/explore-education-statistics-frontend/src/components/Page.tsx

```diff
@@ -4,15 +4,27 @@ import Breadcrumbs, { BreadcrumbsProps } from './Breadcrumbs';
 import PageBanner from './PageBanner';
 import PageFooter from './PageFooter';
 import PageHeader from './PageHeader';
+import PageMeta from './PageMeta';
 
 type Props = {
   children: ReactNode;
   wide?: boolean;
+  pageMeta?: {
```

Comment:
To keep this in sync with changes to `PageMeta`, export the props as `PageMetaProps` and assign the type here

Reply:
Ah, makes perfect sense, will do

Reply:
Done